services/scan/src/importer.ts

```diff
@@ -537,7 +539,10 @@ export class Importer {
       return;
     }
 
-    await this.workspace.store.exportCvrs(writeStream);
+    await pipeline(
+      exportCastVoteRecordsAsNdJson({ store: this.workspace.store }),
+      writeStream
+    );
```

Comment:
We could just raise this up into `central_scanner_app` to match `precinct_scanner_app`, I'm not sure if that consistency is valuable.

Reply:
I'm gonna leave this alone for now.